### ISSUE TITLE
First cell of "create model" not working on ipynb

### ISSUE THREAD

**Issue by masterika** [navidyou/Mask-RCNN-implementation-for-cell-nucleus-detection-executable-on-google-colab-#1]
Hi,
I tried to run all cells here : "https://colab.research.google.com/github/navidyou/Mask-RCNN-implementation-for-cell-nucleus-detection-executable-on-google-colab-/blob/master/mask_RCNN_cell_nucleus_google_colab.ipynb"

This part is no longer working:
# Create model in training mode
model = modellib.MaskRCNN(mode="training", config=config,
                          model_dir=MODEL_DIR)

I think it's because of Tensorflow 2.0 update but I cannot fix it.  Do you have an updated version of the collab notebook?

**Comment by KazuhideMimura** [comment]
I had the same problem and solved by adding following two sentences at the top of the code.

%tensorflow_version 1.x
import tensorflow

**Comment by masterika** [comment]
Thanks! That works. I had to downgrade Keras too.
